tutorials/neow3j-nep17/index.md

```diff
@@ -0,0 +1,405 @@
+        return assetMap.getIntOrZero(key.toByteArray());
+    }
+
+    // Deploy, Update, and Destroy
+
+    @OnDeployment
+    public static void deploy(Object data, boolean update) {
+        if (!update) {
+            throwIfSignerIsNotOwner();
+            int initialSupply = 200_000_000;
+            Storage.put(ctx, totalSupplyKey, initialSupply);
+            assetMap.put(owner.toByteArray(), initialSupply);
+        }
+    }
+
+    public static void update(ByteString script, String manifest) {
+        throwIfSignerIsNotOwner();
+        ContractManagement.update(script, manifest);
+    }
+
+    public static void destroy() throws Exception {
+        throwIfSignerIsNotOwner();
+        ContractManagement.destroy();
+    }
+
+}
+```
+
+## 4. Contract Breakdown
+
+In the following, we're looking at each part of the NEP-17 example contract.
+
+### Imports
+
+The imports show the neow3j devpack classes that are used in the example contract. Check out neow3j devpack's [javadoc](https://javadoc.io/doc/io.neow3j/devpack/latest/index.html) for a full overview of classes and methods that are supported.
+
+```java
+package io.neow3j.examples.contractdevelopment.contracts;
+
+import static io.neow3j.devpack.StringLiteralHelper.addressToScriptHash;
+
+import io.neow3j.devpack.ByteString;
+import io.neow3j.devpack.Contract;
+import io.neow3j.devpack.Hash160;
+import io.neow3j.devpack.Runtime;
+import io.neow3j.devpack.Storage;
+import io.neow3j.devpack.StorageContext;
+import io.neow3j.devpack.StorageMap;
+import io.neow3j.devpack.annotations.DisplayName;
+import io.neow3j.devpack.annotations.ManifestExtra;
+import io.neow3j.devpack.annotations.OnDeployment;
+import io.neow3j.devpack.annotations.Permission;
+import io.neow3j.devpack.annotations.SupportedStandards;
+import io.neow3j.devpack.constants.CallFlags;
+import io.neow3j.devpack.constants.NativeContract;
+import io.neow3j.devpack.contracts.ContractManagement;
+import io.neow3j.devpack.events.Event3Args;
+```
+
+### Contract-specific Information
+
+Annotations on top of the contract's class represent contract-specific information. The following annotations can be used for a contract:
+
+_`@DisplayName`_
+
+Specifies the contract's name. If this annotation is not present, the class name is used for the contrat's name.
+
+_`@ManifestExtra`_
+
+Adds the provided key-value pair information in the manifest's `extra` field. You can also use `@ManifestsExtras` to gather multiple `@ManifestExtra` annotations (results in the same as when using single `@ManifestExtra` annotations).
+
+_`@SupportedStandards`_
+
+Sets the `supportedStandards` field in the manifest.
+
+_`@Permission`_
+
+Specifies, what other contracts and which methods it is allowed to call. By default (i.e., if no permission annotation is set), the contract is not allowed to call any contract. Use `contract = ` to specify what contracts and `methods = ` to specify what methods are allowed.
```

Comment:
```suggestion
Specifies which third-party contracts and methods the smart contract is allowed to call. By default (i.e., if no permission annotation is set) the contract is not allowed to call any contract. Use `contract = ` and `methods = ` to specify, respectively, which contracts and methods are allowed.
```

Reply:
If the same English construction is used for the NEP-11 article, please update there also.